tests/float/math_fun_intbig.py

```diff
@@ -9,3 +9,11 @@
 for fun in (math.ceil, math.floor, math.trunc):
     for x in (-1e25, 1e25):
         print('%.3g' % fun(x))
+
+if 'factorial' in dir(math):
```

Comment:
This will need to go in a separate file called `math_factorial_intbig.py`, with a feature test something like this:
```python
try:
    import math
    math.factorial
except (ImportError, NameError):
    print('SKIP')
    SystemExit
```
Then Travis CI should pass.